smart-contracts/osmosis/contracts/cl-vault/src/helpers/msgs.rs

```diff
@@ -91,14 +95,14 @@ pub fn swap_msg(deps: &DepsMut, env: &Env, params: SwapParams) -> Result<CosmosM
 }
 
 fn osmosis_swap_exact_amount_in_msg(
-    env: &Env,
+    sender: String,
```

Comment:
Changing the type to `Addr` improves type safety

Reply:
done